cmd/oras/blob/fetch.go

```diff
@@ -114,6 +116,14 @@ func fetchBlob(opts fetchBlobOptions) (fetchErr error) {
 		// fetch blob content
 		var rc io.ReadCloser
 		desc, rc, err = oras.Fetch(ctx, src, opts.targetRef, oras.DefaultFetchOptions)
+		vr := content.NewVerifyReader(rc, desc)
+		buf := bytes.NewBuffer(nil)
+		if _, err := io.Copy(buf, vr); err != nil {
+			return err
+		}
```

Comment:
Blobs are usually large and cannot be dumped into memory.

Reply:
Fixed according to our offline discussion.